lib/smart_proxy_pulp_plugin/settings.rb

```diff
@@ -0,0 +1,12 @@
+require 'sinatra'
+require 'smart_proxy_pulp_plugin/pulp_node_plugin'
+require 'smart_proxy_pulp_plugin/pulp_plugin'
+
+module PulpProxy
+  class Settings 
+    def self.settings 
+      # work around until pulp node settings are no longer needed by foreman proxy, as pulp nodes have been removed
+      ::PulpProxy::Plugin.settings.pulp_url ? ::PulpProxy::Plugin.settings : ::PulpNodeProxy::Plugin.settings
```

Comment:
It can't yet.  We currently use it to differentiate between an external pulp server and one that is located on the foreman server itself.  We need to re-work that somehow and then we can remove it.